README.md

```diff
@@ -8,23 +8,36 @@ Repository for developing server side of RFSoC fpga boards
 Qibosoq is a server for integrating [Qick](https://github.com/openquantumhardware/qick) in the [Qibolab](https://github.com/qiboteam/qibolab) ecosystem
 for executing arbitrary pulses sequences on QPUs.
 
-The complete documentation for can be found at [qibo.science/qibosoq/stable](https://qibo.science/qibosoq/stable/)
+The complete documentation for can be found at:
 
-## Installation
+* [qibo.science/qibosoq/stable](https://qibo.science/qibosoq/stable/)
+* [qibo.science/qibosoq/latest](https://qibo.science/qibosoq/latest/)
+
+
+## Installation from pip
```

Comment:
It's nice to have installation instructions on the README, but this is duplicating the information in the docs, so both have to be maintained consistent.

To simplify, I'd consider adding just a link to the docs. What do you think?

Reply:
Seems reasonable, thanks!